packages/testtools/src/schema.ts

```diff
@@ -62,7 +62,7 @@ export function installPackage(pkg: string, dev = false) {
 }
 
 export function normalizePath(p: string) {
-    return p ? p.split(path.sep).join(path.posix.sep) : p;
+    return path.normalize(p);
```

Comment:
Based on some tests of the swr plugin I'm pretty sure this change is incorrect and I should instead ensure the following:

- Paths in zenstack model files use posix separators
- Paths for native filesystem operations are normalized using path.normalize to ensure proper handling of Windows drive prefixes

I have some local changes going in this direction, but I'm still verifying the whole test suite.

Reply:
The path handling was fine as is; I was misdirected by some of the expected-error logging and testcases failing for non-Windows specific reasons (which I'm purposefully not addressing in this branch)